www/sp1/nikd01/index.html

```diff
@@ -9,18 +9,18 @@
     <meta content="width=device-width, initial-scale=1.0" name="viewport">
     <meta content="ie=edge" http-equiv="X-UA-Compatible">
     <title>Modern Web Technologies</title>
-    <link rel="icon" type="image/png" sizes="32x32" href="./assets/favicon/favicon.png">
-    <link rel="apple-touch-icon" sizes="32x32" href="./assets/favicon/favicon.png"/>
+    <link href="./assets/favicon/favicon.png" rel="icon" sizes="32x32" type="image/png">
+    <link href="./assets/favicon/favicon.png" rel="apple-touch-icon" sizes="32x32"/>
     <link href="./assets/css/main.css" rel="stylesheet">
 </head>
 <body>
 <!--
 - html entity >
-- zvyrazneni aktivni polozky menu
-- externi odkaz _blank
-- odkaz na github bez uzivatele
-- chybi cela jedna stranka (minimalni pocet stranek je 5)
-- chybi verze pro tisk
+- zvyrazneni aktivni polozky menu (opraveno)
+- externi odkaz _blank (opraveno)
+- odkaz na github bez uzivatele (opraveno)
+- chybi cela jedna stranka (minimalni pocet stranek je 5) (opraveno - pridana stranka full stack)
+- chybi verze pro tisk (opraveno, ale bohuzel nejde ve verzi pro tisk zobrazovat gradient)
```

Comment:
jak myslite nejde zobrazovat gradient? 
![image](https://user-images.githubusercontent.com/20724910/211046405-9c167bdf-c721-4109-a8e7-6b410b48fdb1.png)

na verzi pro tisk hlavne vyreste plytvani papirem

Reply:
I forgot to check the 'background graphics' checkbox, that's why these gradients weren't displayed.
Margins and font-sizes for the print version were also updated in the new commit.